packages/gas-fee-controller/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.1]
+### Changed
+- Expose `getProviderAndBlockTracker` instead of `provider`([#1091](https://github.com/MetaMask/core/pull/1091))
```

Comment:
this doesn't seem quite right... I guess we need any entry here? Even if all that changed was [how we imported a type?](https://github.com/MetaMask/core/pull/1091/files#diff-e81458d184065326908e9ebcd6f7fa3bd02d5f015f258caab108b820c3304251L306)

Reply:
Yeah, any suggestion for a change?

Reply:
"Adjust types to align with new version of NetworkController"?

Reply:
yeah I think that's probably fine. @mcmire would probably be better at wording this but I know you're trying to move this along quickly! So that works for me.